src/platform/bitboxbase/leds.c

```diff
@@ -15,33 +15,69 @@
 #include "leds.h"
 #include "bitboxbase_pins.h"
 
-static void _turn_small_led(int led, bool level)
+#include <stdint.h>
+static uint8_t small_leds[] = {PIN_LED_SMALL(0),
+                               PIN_LED_SMALL(1),
+                               PIN_LED_SMALL(2),
+                               PIN_LED_SMALL(3),
+                               PIN_LED_SMALL(4)};
+
+void turn_small_led(int led, bool level)
 {
     gpio_set_pin_level(led, level);
 }
 
-static void _turn_big_led(int led, bool level)
+static void _turn_big_led_component(uint8_t pin, bool level)
 {
-    gpio_set_pin_level(led, !level);
+    gpio_set_pin_level(pin, !level);
 }
 
-typedef enum { LED_WHITE, LED_RED, LED_GREEN, LED_BLUE } led_color_t;
+static const uint8_t big_leds[2][3] = {{PIN_BLED0_R, PIN_BLED0_G, PIN_BLED0_B},
+                                       {PIN_BLED1_R, PIN_BLED1_G, PIN_BLED1_B}};
 
-static const uint8_t big_leds[] =
-    {PIN_BLED0_R, PIN_BLED0_G, PIN_BLED0_B, PIN_BLED1_R, PIN_BLED1_G, PIN_BLED1_B};
+void turn_big_led(int led, led_color_t color)
```

Comment:
`turn_big_led` should be prefixed with `leds` to be consistent. We always prefix non-static functions with the filename.

Reply:
Fixed.